src/linkpresentation.cs

```diff
@@ -0,0 +1,99 @@
+//
+// LinkPresentation C# bindings
+//
+// Authors:
+//	TJ Lambert  <t-anlamb@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation All rights reserved.
+//
+
+#if MONOMAC
+using AppKit;
+using UIView = AppKit.NSView;
+#else
+using UIKit;
+#endif
+
+using System;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace LinkPresentation {
+
+	[ErrorDomain ("LPErrorDomain")]
+	[NoWatch]
```

Comment:
you do not have to remove them - they just won't do anything if not part of the watchOS build (from `src/frameworks.sources`)

Reply:
Oh right good point!